x-pack/plugins/maps/public/shared/layers/sources/es_search_source/update_source_editor.js

```diff
@@ -62,45 +63,32 @@ export class UpdateSourceEditor extends Component {
     this.setState({ fields: indexPattern.fields });
   }
 
-  onTooltipPropertiesSelect = (propertyNames) => {
+  _onTooltipPropertiesSelect = (propertyNames) => {
     this.props.onChange({ propName: 'tooltipProperties', value: propertyNames });
   };
 
-  onFilterByMapBoundsChange = event => {
+  _onFilterByMapBoundsChange = event => {
     this.props.onChange({ propName: 'filterByMapBounds', value: event.target.checked });
   };
 
   render() {
     return (
       <Fragment>
-        <EuiFormRow
-          label={
-            i18n.translate('xpack.maps.source.esSearch.fieldsLabel', {
-              defaultMessage: `Fields to display in tooltip`
-            })
-          }
-        >
-          <MultiFieldSelect
-            placeholder={i18n.translate('xpack.maps.source.esSearch.fieldsPlaceholder', {
-              defaultMessage: `Select fields`
-            })
-            }
-            value={this.props.tooltipProperties}
-            onChange={this.onTooltipPropertiesSelect}
-            fields={this.state.fields}
-          />
-        </EuiFormRow>
+        <TooltipSelector
+          tooltipProperties={this.props.tooltipProperties}
```

Comment:
Lets rename `tooltipProperties` to `value` to be more consistent with most input elements where the value is called value